rules/change-date-of-existing-commit/rule.md

```diff
@@ -0,0 +1,85 @@
+---
+type: rule
+seoDescription: Learn how to change the date of an existing commit
+archivedreason: 
+title: Do you know how to change the date of an existing commit?
+guid: f29b94c2-14be-4900-8734-924c9a576f4e
+uri: change-date-of-existing-commit
+created: 2024-07-29T11:06:33.0000000Z
+authors:
+  - title: Jeoffrey Fischer
+    url: https://ssw.com.au/people/jeoffrey-fischer
+related:
+- clean-git-history
+redirects: []
+
+---
+
+Updating commit information can be essential for maintaining accurate project history or correcting errors.
+Whether you need to change a commit date for clarity, compliance, or other reasons, you have a couple of methods at your disposal.
+This rule outlines how to change the date of an existing commit using both a manual CLI approach and an automated script.
+
+<!--endintro-->
+
+# Method 1 – Use CLI
```

Comment:
Hey @jeoffreyfischer 

These should be H2 (the main title is already H1)

- [ ] 1. Please fix

Reply:
Done